src/styles/partials/components/_search.scss

```diff
@@ -17,9 +17,9 @@
 }
 
 .dg_search-input {
-    padding: $padding-small;
-    border: 2px solid #000;
-    font-size: 20px;
-    font-family: "Montserrat",sans-serif;
-    width: 100%;
+  padding: $padding-small;
+  border: 2px solid #000;
+  font-size: 20px;
```

Comment:
Recommend using variable for 20px size if available.

Reply:
done